server.go

```diff
@@ -241,6 +251,14 @@ func NewServer(opts ...ServerOption) (*Server, error) {
 	s.Cluster.Logger = s.logger
 	s.Cluster.Holder = s.Holder
 
+	// Initialize translation database.
+	s.TranslateFile = NewTranslateFile()
+	s.TranslateFile.Path = filepath.Join(path, "keys")
```

Comment:
This `keys` file in the root of the data directory prevents having an index called `keys`. We could continue with the `.keys` pattern, but making everything a hidden file seems wrong. A better solution may be to have indexes reside in a subdirectory to avoid collisions. Just noting here that we should ticket this to be addressed later.